### PR TITLE
[CoreBundle] Fix coupon association

### PR DESCRIPTION
**Pull request by inssein**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes https://github.com/Sylius/Sylius/issues/4456
| License         | MIT
| Doc PR          | --

I know @gonzalovilaseca said he would submit a PR, but I just came across this and needed the fix.

### CODE REVIEW

**Comment by michalmarcinkowski** [comment]
:+1: but migration file is missing :)

**Comment by inssein** [comment]
@michalmarcinkowski ah good catch, I even generated one, but forgot to commit since I did this over the GitHub UI. Will add it when I get a chance.

**Comment by inssein** [comment]
@michalmarcinkowski should be good to go now.

**Comment by pjedrzejewski** [comment]
Restarted the build on PHP7.

**Comment by inssein** [comment]
@pjedrzejewski failed again :( but it's not only just this PR

**Comment by inssein** [comment]
ping @pjedrzejewski - is there anything else I need to do?

File: app/migrations/Version20160310181749.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Sylius\Migrations;
```
Comment (Zales0123) [inline app/migrations/Version20160310181749.php]: I think there should be license block here - @pjedrzejewski did we agreed about this?

**Comment by michalmarcinkowski** [comment]
Thanks Hussein! :+1:

File: app/migrations/Version20160310181749.php (hunk 1 above)
Comment (michalmarcinkowski) [inline app/migrations/Version20160310181749.php]: Several migration files don't have this block, so I suppose we can add it to all of them in a separate PR.

Comment (Zales0123) [inline app/migrations/Version20160310181749.php]: :+1:

**Pull request merged**